pkg/apis/descheduler/v1/types_descheduler.go

```diff
@@ -71,10 +71,20 @@ type ProfileCustomizations struct {
 	// the default exclusion of all openshift-*, kube-system and hypershift namespaces
 	Namespaces Namespaces `json:"namespaces"`
 
-	// LowNodeUtilizationThresholds enumerates predefined experimental thresholds
+	// DevLowNodeUtilizationThresholds enumerates predefined experimental thresholds
 	// +kubebuilder:validation:Enum=Low;Medium;High;""
 	DevLowNodeUtilizationThresholds *LowNodeUtilizationThresholdsType `json:"devLowNodeUtilizationThresholds"`
 
+	// DevEnableSoftTainter enables SoftTainter alpha feature.
+	// The EvictionsInBackground alpha feature is a subject to change.
+	// Currently provided as an experimental feature.
+	DevEnableSoftTainter bool `json:"devEnableSoftTainter"`
```

Comment:
Would you please extend the README.md with the new tuning options as well? `DevEnableSoftTainter` is a good example of disabling the controller in case users decide to provide their own implementation. Yet, still limited in the scope of the new profile only.

Reply:
Done